packages/api/lib/rulesHelpers.js

```diff
@@ -590,8 +593,8 @@ async function addRule(item, payload) {
 /**
  * Checks if record is valid
  *
- * @param {RuleRecord} rule - Rule to check validation
- * @returns {void}          - Returns if record is valid, throws error otherwise
+ * @param {any} rule - Object to validate as a Rule Record validation
+ * @returns {RuleRecord}          - Returns if record is valid, throws error otherwise
```

Comment:
```suggestion
 * @param {any} rule     - Object to validate as a Rule Record validation
 * @returns {RuleRecord} - Returns if record is valid, throws error otherwise
```